configuration/introduction.md

```diff
@@ -1,47 +1,119 @@
-| `electronOptions`  | An optional array of Electron (browser) options. This allows configuration of e.g. the browser screen size and position (example: `electronOptions: { fullscreen: false, width: 800, height: 600 }`). Kiosk mode can be enabled by setting `kiosk: true`, `autoHideMenuBar: false` and `fullscreen: false`. More options can be found [here](https://github.com/electron/electron/blob/master/docs/api/browser-window.md).                                                                                                                                |
-| `electronSwitches` | An optional array of Electron switches. This allows configuration of electron app itself. <br> This properties will not affect the `serveronly` mode. Usually normal `MM` users don't need this property, but if you are a hard-core hacker, you might need this to handle Electron itself over `MagicMirror` provides. More options can be found [here](https://www.electronjs.org/docs/latest/api/command-line-switches) (Not all available switches are described there.)<br>example:`electronSwitches:["enable-transparent-visuals", "disable-gpu"];` |
-| `customCss`        | The path of the `custom.css` stylesheet. The default is `css/custom.css`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-
-### Environment variables
-
-As the `config.js` is javascript code there is no direct way to provide
-environment variables. This feature was requested by some users, e.g. with
-environment variables it is easier to share the configuration with others
-(otherwise you have to delete all private data as e.g. api keys). There was a
-longer
-[discussion on github](https://github.com/MichMich/MagicMirror/issues/1756)
-where you can read the full story.
-
-#### `config.js` and `config.js.template`
-
-You can provide a `config.js.template` instead of a `config.js` file. The
-difference is, that you can use environment variables in the
-`config.js.template`. When starting MagicMirror² a `config.js` is created from
-`config.js.template` and the variables are resolved.
+
+
+### Extremely Basic Configuration of MagicMirror²: 
```

Comment:
I suggest to change this heading to "Minimum Configuration of MagicMirror²".

Reply:
of course, it's not the minimum. I decided to go with the most basic 'useful' configuration. In playing with the mirror for documentation, you CAN get away with 
```
let config = { }
```

It'll run, it won't be happy, but it'll run.

Reply:
You are right. Somehow the word "Extremely" just bothers me. But maybe I just have a strange feeling for the language

Reply:
Thinking on it, you're right. I'll strike that.

Reply:
Nice! Thanks.